src/components/medialibrarycreator/medialibrarycreator.js

```diff
@@ -42,11 +42,9 @@ define(["loading", "dialogHelper", "dom", "jQuery", "components/libraryoptionsed
     }
 
     function getCollectionTypeOptionsHtml(collectionTypeOptions) {
-        return collectionTypeOptions.filter(function(i) {
-            return i.isSelectable
```

Comment:
Should the functionality still exist / can isSelectable end up false? If so I would recommend adding `i.isSelectable !== false` back in :)
Edit: plus a comment so it doesn't regress again